datahub-web-react/src/app/entity/shared/containers/profile/sidebar/Ownership/EditOwnersModal.tsx

```diff
@@ -78,10 +78,25 @@ export const EditOwnersModal = ({
     const renderSearchResult = (entity: Entity) => {
         const avatarUrl =
             (entity.type === EntityType.CorpUser && (entity as CorpUser).editableProperties?.pictureLink) || undefined;
+        const isCorpUser = (entity.type as EntityType) === EntityType.CorpUser;
+        const corpUserDepartmentName =
+            (entity.type === EntityType.CorpUser && (entity as CorpUser).properties?.departmentName) || '';
+        const corpUserId = (entity.type === EntityType.CorpUser && (entity as CorpUser).username) || '';
+        const corpUserTitle = (entity.type === EntityType.CorpUser && (entity as CorpUser).properties?.title) || '';
         const displayName = entityRegistry.getDisplayName(entity.type, entity);
+        const concatenatedValue = isCorpUser
```

Comment:
Please use dashes or separators to divide them up..

E.g. "Corp user Id - Corp user title - corp user department name"

When you do this, your logic will have to change to account for the fact that certain fields can be null / missing

Reply:
Separately, I would STRONGLY recommend changing OwnerLabel to take in the additional fields directly, and creating a second <div/> underneath that contains the corpUserId, corpUserTitle, etc. 

If these fields are present we can show the sub-header div, if not we will hide it.

This allows things to be slightly more reusable for Groups if we add department, unique id to a group. Logic can be agnostic of entity type.

Reply:
Addressed